src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -214,15 +214,17 @@ const SignOrExecuteForm = ({
           />
         )}
 
-        <GasParams
-          isExecution={willExecute}
-          isLoading={isEstimating}
-          nonce={advancedParams.nonce}
-          gasLimit={advancedParams.gasLimit}
-          maxFeePerGas={advancedParams.maxFeePerGas}
-          maxPriorityFeePerGas={advancedParams.maxPriorityFeePerGas}
-          onEdit={() => setEditingGas(true)}
-        />
+        {tx && (
```

Comment:
I deliberately didn't add this check and took extra care to handle undefined in the form itself. Because otherwise it jumps whilst creating a tx.

Reply:
There is already a slight jump when creating a transaction though:

![tx](https://user-images.githubusercontent.com/20442784/185905022-c689010b-fe50-4df2-801e-3d33b5244468.gif)

I don't think it makes much sense to add gas parameters when there is no transaction. We aren't creating one when the selected threshold matches that of the current threshold.

We could alternatively add another intermediary UI when there's no `nonce` [here](https://github.com/safe-global/web-core/blob/main/src/components/tx/GasParams/index.tsx#L76-L79) or wrap the `<GasParams>` in a [transition component](https://mui.com/material-ui/transitions/#collapse) to make things smoother. What do you think?